protocols/bizcoin/blockchain/blkparser/tx.go

```diff
@@ -0,0 +1,127 @@
+// Basically adapation from the file at https://github.com/tsileo/blkparser
+package blkparser
+
+import (
+	"encoding/binary"
+
+	"github.com/btcsuite/btcd/chaincfg"
+	"github.com/btcsuite/btcd/txscript"
+)
+
+type Tx struct {
+	Hash     string
+	Size     uint32
+	LockTime uint32
+	Version  uint32
+	TxInCnt  uint32
+	TxOutCnt uint32
+	TxIns    []*TxIn
+	TxOuts   []*TxOut
+}
+
+type TxIn struct {
+	InputHash string
+	InputVout uint32
+	ScriptSig []byte
+	Sequence  uint32
+}
+
+type TxOut struct {
+	Addr     string
+	Value    uint64
+	Pkscript []byte
+}
+
+func ParseTxs(txsraw []byte) (txs []*Tx, err error) {
```

Comment:
ParseTxs is used to parse the transactions  (from a .dat file to create a Block struct)